tomster-player/app/router.js

```diff
@@ -6,8 +6,10 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('album', { path: '/albums/:album_id' }, function() {
-    this.route('comment');
+  this.route('albums', function() {
+    this.route('album', { path: ':album_id' }, function() {
+      this.route('comment');
+    });
```

Comment:
This is the important change here, as well as the outlet in the `albums` template.